index.html

```diff
@@ -43,6 +43,7 @@
         <li><a href="/user/build-timeouts">My builds are timing out, what can I do?</a></li>
         <li><a href="/user/repository-providers">What repository providers can I use?</a></li>
         <li><a href="/user/pricing/">Why can't I find pricing information?</a></li>
+        <li><a href="/user/ci-environment-trusty-beta/">Can I use Ubuntu Trusty for my builds?"</a><li>
```

Comment:
My proposal is using ``/user/trusty-ci-environment`` like the one proposed by @solarce and the one we have for osx:  ``/user/osx-ci-environment/``

I think that Trusty deserves its own place at the moment, but I'd also mention it at ``/user/ci-environment/`` in: [Virtualization Environments](http://docs.travis-ci.com/user/ci-environment/#Virtualization-environments) and it's differences at [Environment common to all VM images](http://docs.travis-ci.com/user/ci-environment/#Environment-common-to-all-VM-images.)